vendor/github.com/tidwall/gjson/SYNTAX.md

```diff
@@ -0,0 +1,342 @@
+# GJSON Path Syntax
+
+A GJSON Path is a text string syntax that describes a search pattern for quickly retreiving values from a JSON payload.
```

Comment:
```suggestion
A GJSON Path is a text string syntax that describes a search pattern for quickly receiving values from a JSON payload.
```

Reply:
This is in vendor, we don't manage the code here